content/spin/v2/cli-reference.md

```diff
@@ -1877,6 +2167,28 @@ OPTIONS:
 
 {{ blockEnd }}
 
+{{ startTab "v2.7"}}
+
+<!-- @selectiveCpy -->
+
+```console
+$ spin plugins list --help   
```

Comment:
Interesting, I wonder if there is a bug in some part of the toolkit scripts.  I was double-checking this output per the diff you pasted and the output in 2.7 appears to be different:
```
spin-plugins-list 
List available or installed plugins

USAGE:
    spin plugins list [OPTIONS]

OPTIONS:
        --all                List all versions of plugins. This is the default behaviour
        --filter <FILTER>    Filter the list to plugins containing this string
    -h, --help               Print help information
        --installed          List only installed plugins
        --summary            List latest and installed versions of plugins

```